### PR TITLE
Notification drawer tweaks

### PR DESCRIPTION
**Pull request by epwinchell**
Jira: https://issues.redhat.com/browse/RHCLOUD-28554

Old
Screenshot 2024-02-27 at 3 41 06 PM

New
Screenshot 2024-02-28 at 4 49 02 PM

### CODE REVIEW

File: src/components/NotificationsDrawer/DrawerPanelContent.tsx [hunk 1]
```diff
@@ -103,43 +101,32 @@ const DrawerPanelBase = ({ innerRef }: DrawerPanelProps) => {
...
+    <DropdownItem value={0} key="action" description="Actions"></DropdownItem>,
     <DropdownItem key="read all" onClick={onMarkAllAsRead} isDisabled={notifications.length === 0}>
       Mark visible as read
     </DropdownItem>,
     <DropdownItem key="unread all" onClick={onMarkAllAsUnread} isDisabled={notifications.length === 0}>
       Mark visible as unread
     </DropdownItem>,
     <Divider key="divider" />,
+    <DropdownItem value={0} key="action" description="Quick links">
+      ,
```
Comment (aferd) [inline src/components/NotificationsDrawer/DrawerPanelContent.tsx:113]: is this comma needed?

File: src/components/NotificationsDrawer/DrawerPanelContent.tsx [hunk 2]
```diff
@@ -103,43 +101,32 @@ const DrawerPanelBase = ({ innerRef }: DrawerPanelProps) => {
...
+    </DropdownItem>,
```
Comment (aferd) [inline src/components/NotificationsDrawer/DrawerPanelContent.tsx:114]: and this one?

File: src/components/NotificationsDrawer/DrawerPanelContent.tsx (hunk 1 above)
Comment (epwinchell) [reply src/components/NotificationsDrawer/DrawerPanelContent.tsx:113]: Yes, it throws an error without the commas . The other dropdownitems have them, as well.

Comment (aferd) [reply src/components/NotificationsDrawer/DrawerPanelContent.tsx]: ah, I see, I think this one might be misplaced as it shows in the image above the "Quick Links" text

Comment (epwinchell) [reply src/components/NotificationsDrawer/DrawerPanelContent.tsx]: good catch!

File: src/components/NotificationsDrawer/DrawerPanelContent.tsx [hunk 3]
```diff
@@ -103,43 +101,30 @@ const DrawerPanelBase = ({ innerRef }: DrawerPanelProps) => {
   };
 
   const dropdownItems = [
+    <DropdownItem value={0} key="actions" description="Actions"></DropdownItem>,
```
Comment (fhlavac) [inline src/components/NotificationsDrawer/DrawerPanelContent.tsx:104]:
```suggestion
    <DropdownItem value={0} key="actions" description="Actions"/>,
```

File: src/components/NotificationsDrawer/DrawerPanelContent.tsx [hunk 4]
```diff
@@ -103,43 +101,30 @@ const DrawerPanelBase = ({ innerRef }: DrawerPanelProps) => {
...
     <DropdownItem key="read all" onClick={onMarkAllAsRead} isDisabled={notifications.length === 0}>
       Mark visible as read
     </DropdownItem>,
     <DropdownItem key="unread all" onClick={onMarkAllAsUnread} isDisabled={notifications.length === 0}>
       Mark visible as unread
     </DropdownItem>,
     <Divider key="divider" />,
+    <DropdownItem value={0} key="quick links" description="Quick links"></DropdownItem>,
```
Comment (fhlavac) [inline src/components/NotificationsDrawer/DrawerPanelContent.tsx:112]:
```suggestion
    <DropdownItem value={0} key="quick links" description="Quick links"/>,
```

**Comment by fhlavac** [comment]
Looks like the tests need to be updated, otherwise looks good!

File: src/components/NotificationsDrawer/DrawerPanelContent.tsx [hunk 5]
```diff
@@ -103,43 +101,30 @@ const DrawerPanelBase = ({ innerRef }: DrawerPanelProps) => {
   };
 
   const dropdownItems = [
+    <DropdownItem value={0} key="actions" description="Actions" />,
```
Comment (Hyperkid123) [inline src/components/NotificationsDrawer/DrawerPanelContent.tsx:104]: is the `value` prop necessary here?

**Review by Hyperkid123 — APPROVED** [review]

**Pull request merged**